### ISSUE TITLE
Override property in test.yaml

### ISSUE THREAD

**Issue by johnnynunez** [ashleve/lightning-hydra-template#282]
Hi,
How could I override a property in test.yaml?

For example a property for datamodule. In experiments works fine but in test.yaml, I can't override a property

**Comment by ashleve** [comment]
@johnnynunez Hi, it should work if you remove the `_self_` from defaults or declare it after all the other subconfigs.
```yaml
# specify here default evaluation configuration
defaults:
  - _self_  #  
  - ...
```
The order of config modules in defaults actually determines the order in which configs get overriden, meaning that `_self_` at the beginning causes the `test.yaml` to be overriden by other subconfigs.

**Comment by johnnynunez** [comment]
@ashleve oh thank you! This works. 
I have another question, is it possible to track the learning rate? Because, I would like see the lr in x epoch with weight decay

**Comment by ashleve** [comment]
@johnnynunez there's a callback for that
https://pytorch-lightning.readthedocs.io/en/latest/api/pytorch_lightning.callbacks.LearningRateMonitor.html
